__tests__/app.test.js

```
@@ -402,4 +402,172 @@ test("returns status 400 when passed not an object", () => {
+                                    topic: expect.any(String),
+                                    created_at: expect.any(String),
+                                    votes: expect.any(Number),
+                                    comment_count: expect.any(Number)
+                                })
+                            )
+                        })
+                    })
+                    })
+
+                    test("returns an articles array of article objects | SAD PATH | INVALID sort_by (then default sort by time) ?sort_by=hello", () => {
+
+                        return request(app)
+                        .get("/api/articles?sort_by=hello")
+                        .expect(200).then(({body : {articles}}) => {
+                
+                            expect(articles).toHaveLength(12)
+                            expect(articles).toBeSortedBy('created_at',{descending : true})
+                
+                            articles.forEach((article) => {
+                                expect(article).toEqual(
+                                    expect.objectContaining({
+                                        author : expect.any(String),
+                                        title: expect.any(String),
+                                        article_id: expect.any(Number),
+                                        topic: expect.any(String),
+                                        created_at: expect.any(String),
+                                        votes: expect.any(Number),
+                                        comment_count: expect.any(Number)
+                                    })
+                                )
+                            })
+                        })
+                        })
+                        
+                        test("returns an articles array of article objects | SAD PATH | INVALID order (then default desc, while default sort by time when not specified) ?order=yummyfood", () => {
```

Im not sure how this is different to the above one, but both should return an error